### PR TITLE
add loco framework template

### PR DESCRIPTION
**Pull request by kaplanelad**
## Description of change
Added a sample template for [Loco](https://loco.rs/):

## How has this been tested? (if applicable)
Deployed the Loco website using the shuttle CLI.

### CODE REVIEW

File: loco/hello-world/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- **Task**: Ad-hoc functionalities to handle specific aspects of your application
+- **Tests**: Testing facility and examples
+- And More Loco Features
+
+## Quick Start
+
+To launch your app, simply use the following command:
+
+```sh
+$ cargo loco start
```
Comment (jonaro00) [inline loco/hello-world/README.md:15]: The preferred way when Shuttle has been added would be 
```suggestion
cargo shuttle run
```

File: loco/hello-world/.devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Loco",
+    "dockerComposeFile": "docker-compose.yml",
+    "service": "app",
+    "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
+    "forwardPorts": [
+        3000
```
Comment (jonaro00) [inline loco/hello-world/.devcontainer/devcontainer.json:7]: cargo-shuttle runs on 8000 by default. Should that be added here?

**Review by jonaro00 — COMMENTED** [review]
Is the `.cargo/config.toml` with the cli alias missing?

File: loco/hello-world/README.md (hunk 1 above)
Comment (kaplanelad) [reply loco/hello-world/README.md]: changed

File: loco/hello-world/.devcontainer/devcontainer.json (hunk 2 above)
Comment (kaplanelad) [reply loco/hello-world/.devcontainer/devcontainer.json]: change

**Pull request merged**